roles/build_edge_installer/tasks/build-edge-installer.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+- name: Create a blueprint
+  infra.osbuild.create_blueprint:
+    dest: "{{ builder_blueprint_src_path }}-installer"
+    name: "{{ builder_blueprint_name }}-installer"
+  register: blueprint_output
+
+- name: Push the blueprint into image builder
+  infra.osbuild.push_blueprint:
+    src: "{{ builder_blueprint_src_path }}-installer"
+
+- name: Start compose
+  infra.osbuild.start_compose:
+    blueprint: "{{ builder_blueprint_name }}-installer"
+    compose_type: "edge-installer"
+    ostree_url: "http://{{ ansible_default_ipv4.address }}/{{ builder_blueprint_name }}/repo"
+  register: compose_start_out
+
+- name: Wait for compose to finish
+  infra.osbuild.wait_compose:
+    compose_id: "{{ compose_start_out['result']['build_id'] }}"
+
+- name: Create tmp directory for blueprint
+  ansible.builtin.file:
+    path: "/tmp/{{ builder_blueprint_name }}-installer"
+    mode: 0755
+    state: directory
+
+- name: Export the compose artifact
+  infra.osbuild.export_compose:
+    compose_id: "{{ compose_start_out['result']['build_id'] }}"
+    dest: "/tmp/{{ builder_blueprint_name }}-installer/{{ builder_blueprint_name }}-{{ blueprint_output['current_version'] }}.{{ compose_start_out['result']['output_type'] }}" # noqa yaml[line-length]
+
+- name: Copy iso to web dir
+  ansible.builtin.copy:
+    src: "/tmp/{{ builder_blueprint_name }}-installer/{{ builder_blueprint_name }}-{{ blueprint_output['current_version'] }}.{{ compose_start_out['result']['output_type'] }}" # noqa yaml[line-length]
+    dest: /var/www/html/iso
+    remote_src: true
+    mode: '0644'
```

Comment:
0644 permissions are blocking this dir from being served on the server

Reply:
I _think_ '0644' is still fine but I'm betting we should set owner/group to `apache:apache`, let me test that